### ISSUE TITLE
Problem with gdbsource

### ISSUE THREAD

**Issue by embiuw** [kaskr/adcomp#302]
Hi,
I am having trouble getting gdbsource to work on my Windows machine (see sys info below). I do not have an easily reproducible example at the moment, but here are some details on what I did. 

I followed the instructions under 'Windows debugging' here:
https://github.com/kaskr/adcomp/wiki/Windows-installation
Below are the commands and output. As far as I can see all looks ok:

______________________________________________________________
> shell("where g++")
C:\Rtools\mingw_64\bin\g++.exe
C:\Rtools\mingw_32\bin\g++.exe

> shell("where gdb")
C:\Rtools\bin\gdb.exe
C:\Rtools\mingw_64\bin\gdb.exe
C:\Rtools\mingw_32\bin\gdb.exe

> shell("where Rterm")
C:\Program Files\R\R-3.6.0\bin\x64\Rterm.exe
______________________________________________________________

When I run gdbsource ( with interactive = TRUE) on my R script, I get the following message in the gdb shell:

"C:\Program Files\R\R-3.6.0\bin\x64/Rterm.exe": not in executable format: File format not recognized
No symbol table is loaded.  Use the "file" command.

Does anybody have any idea what might b causing this error? I am not an administrator, but have to open Rstudio as local admin to run TMB.

Thanks in advance!

Martin

TMB Version: 1.7.15
R Version: 3.6.0 (2019-04-26)
Operating System: 
sysname        "Windows"      
release        "10 x64"       
version        "build 17134"  
machine        "x86-64"       
login          "Administrator"
user           "Administrator"
effective_user "Administrator"

**Comment by kaskr** [comment]
@embiuw 
The link you provided states that:  *The first occurrence is what matters i.e. must be the 64 bit version*. Your output of `shell("where gdb")` doesn't seem to have the 64 bit version first?

**Comment by embiuw** [comment]
Yes, I was wondering if that could be the reason. I tried putting the fixwinpath function in my R script, but it made no difference. I have't looked closely at that functioj to see if if also changes the order of the gdb repositories it searches through? I will check that more closely. Thanks!
